src/utils/manager.py

```diff
@@ -237,3 +293,56 @@ def umount(mountpoint: Union[str, os.PathLike], force: bool = False) -> None:
     except subprocess.TimeoutExpired as e:
         _logger.error(f"{e} Reason:\n{e.stderr}")
         raise Error(f"Failed to unmount {mountpoint} failed after {e.timeout:.0f}s")
+
+
+def _trigger_autofs() -> None:
```

Comment:
Can you explain to me why this function is necessary and why you used `findmnt`? Is it more just that you're trying to trigger the mount so that you can scan the contents?

Reply:
Yeah, `nfs` points handled by `autofs` don't appear on `/proc/mounts` unless you trigger a mount beforehand.

I used `findmnt` because the function doesn't really need all the info provided by the regex, so I figured it was easier to just filter out all the unneeded filesystems and data with the `-t` and `-o` flags from `findmnt`. However, this can also be done by reading `/proc/mounts` manually if we want to avoid using a subprocess for this.

Reply:
Removed `findmnt` in the new changes.